test/helpers/loans.ts

```diff
@@ -1,3 +1,8 @@
+// external packages
```

Comment:
don't comment the imports like this, `eslint` will auto sort imports based on if its an package or local file